src/ophys_etl/workflows/pipeline_modules/segmentation.py

```diff
@@ -88,13 +88,20 @@ def save_rois_to_db(
 
         for roi in rois:
             # 1. Add ROI
-            mask = roi["mask_matrix"]
+            mask = roi['mask_matrix']
+            motion_border = False
+            if 'motion_border' in roi['exclusion_labels']:
+                motion_border = True
+            if 'small_size' in roi['exclusion_labels']:
+                small_size = False
```

Comment:
u sure about this?

Reply:
Ah, nice catch. No idea why I flipped it to false.